docs/Laravel.md

```diff
@@ -99,7 +94,9 @@ hooks:
 Since we are writing the config cache to disk, all the paths in the config file will be resolved. Since those paths do not exist on our machine (they exist on the lambda environment only) we will have fake paths in the cached config. This is a problem in `config/views.php` because `realpath()` is used. We need to remove the `realpath()` call:
 
 ```diff
--    'compiled' => realpath(storage_path('framework/views')),
+-    'compiled' => env(
+-    'VIEW_COMPILED_PATH',
+-    realpath(storage_path('framework/views'))
 +    'compiled' => storage_path('framework/views'),
```

Comment:
I believe we can now remove this whole bit because of the `VIEW_COMPILED_PATH` variable in `.env`?

Reply:
Yes you are correct, sorry for this was using online editor. I will review all in one shot and submit a better version.